microshift_release_notes/microshift-4-18-release-notes.adoc

```diff
@@ -72,6 +72,13 @@ With this release, you can delete or upgrade the Kustomize manifest resources. F
 //[id="microshift-4-18-support_{context}"]
 //=== Support
 
+[id="microshift-4-18-backup_and_restore_{context}"]
+=== Backup and restore
+
+[id="microshift-4-18-autorecovery-from-manual-backup_{context}"]
+==== Auto recovery from manual backups now documented
+With this release, you can automatically restore data from manual backups when {microshift-short} fails to start by using the `auto-recovery` feature. For more information, see xref:../microshift_backup_and_restore/microshift-auto-recover-manual-backup.adoc#microshift-auto-recover-manual-backup[Auto recovery from manual backups].
```

Comment:
"auto" is a [prefix](https://www.ibm.com/docs/en/ibm-style?topic=grammar-prefixes) rather than a word on its own. This should probably be "automated recovery". 

I recognize that has impact on the linked content in the main feature docs as well. Looking at the feature docs, it looks like there is an `--auto-recovery` flag that you can specify to "automatically restore data". So probably something like "the automated recovery feature" is more in line with our style guide.

I will merge this for now but recommend updating the feature docs and this announcement along these lines:
```suggestion
[id="microshift-4-18-autorecovery-from-manual-backup_{context}"]
==== Automated recovery from manual backups now documented
With this release, you can automatically restore data from manual backups when {microshift-short} fails to start by using the `auto-recovery` feature. For more information, see xref:../microshift_backup_and_restore/microshift-auto-recover-manual-backup.adoc#microshift-auto-recover-manual-backup[Automated recovery from manual backups].
```